Update to .net 6 and add Swagger Filter

kennethlong opened the pull request:
I updated to .Net 6, added a Swagger filter to make Pagination and Sort work with Swagger page, updated nuget package version, updated documentation, updated packages to latest released versions that support .net 6.

kennethlong committed:
Update to .net 6 and add Swagger Filter

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/jhipster/jhipster-net?pullRequest=10) <br/>All committers have signed the CLA.

nicolas63 commented on src/JHipsterNet.Web/Pagination/Swagger/PageableModelFilter.cs:
```diff
@@ -0,0 +1,45 @@
+                return;
+            }
+
+            operation.Parameters.Add(new OpenApiParameter {
```
Maybe we can add a little description in the parameter

nicolas63 commented:
Can you sign the CLA and also update the .tools-versions ?

kennethlong committed:
Added comments

kennethlong commented:
I updated .tools-versions in the first commit, let me know if I need to change that.
I committed this from my work machine which has my work address, I added that work address to my account, but CLA is still saying there is a problem.  Let me check everything again.

kennethlong commented:
It just took like 10 minutes for everything to sync up, looks good now!

nicolas63 commented:
it's ok for me
